core/store/src/main/java/mil/nga/giat/geowave/core/store/adapter/AvroAdapterPersistenceEncoding.java

```diff
@@ -0,0 +1,42 @@
+package mil.nga.giat.geowave.core.store.adapter;
+
+import mil.nga.giat.geowave.core.index.ByteArrayId;
+import mil.nga.giat.geowave.core.store.data.PersistenceEncoding;
+import mil.nga.giat.geowave.core.store.data.PersistentDataset;
+import mil.nga.giat.geowave.core.store.data.PersistentValue;
+import mil.nga.giat.geowave.core.store.index.CommonIndexValue;
+
+/**
+ * This is an implementation of persistence encoding that also contains all of
+ * the extended data values used to form the native type supported by this
+ * adapter. It does not contain any information about the entry in a particular
+ * index and is used when writing an entry, prior to its existence in an index.
+ */
+public class AvroAdapterPersistenceEncoding extends
```

Comment:
Do we need this class?  The implementation is the same as AdapterPersistenceEncoding.  I also don't see this class being used anywhere in this pull request.